pyproject.toml

```diff
@@ -21,6 +21,11 @@ extend-exclude = '''
 
 [tool.pytest.ini_options]
 addopts = "-ra -v"
+# XXX: uncomment if in need off all the captured logs with timing and stuff
```

Comment:
```suggestion
# Note: uncomment to see the  logs with timing and data
```

However, this isn't actually commented out? So the comment doesn't match reality?

Should this instead default to `log_cli = false` and can be flipped to true? Rather than commenting/uncommenting?